.travis.yml

```diff
@@ -79,11 +79,17 @@ jobs:
         - bin/test-console-check
         - yarn build-storybook
     - stage: Test
-      name: "E2E"
+      name: "E2E Node 1"
       script:
         - bundle exec rails db:create db:schema:load assets:precompile
         - yarn cypress install
         - bin/e2e-ci
+    - stage: Test
+      name: "E2E Node 2"
```

Comment:
Added a new build node as discussed. cc: @maestromac. Mac, let me know if you're OK with the `name`.

Reply:
~how bout E2E2?~

Sounds good :+1:

Reply:
Since we have `seededFlows`  and `nonSeededFlows` maybe the node names should represent this, i.e. `E2E Seeded` and `E2E Non-seeded`.